code/game/say.dm

```diff
@@ -36,8 +36,15 @@ GLOBAL_LIST_INIT(freqtospan, list(
 
 /atom/movable/proc/send_speech(message, range = 7, obj/source = src, bubble_type, list/spans, datum/language/message_language = null, list/message_mods = list())
 	var/rendered = compose_message(src, message_language, message, , spans, message_mods)
+	var/list/show_overhead_message_to = list()
 	for(var/atom/movable/AM as() in get_hearers_in_view(range, source))
+		if(ismob(AM))
```

Comment:
Check for clients not mobs

Reply:
Clients arent returned by get hearers in view since they are not something that physically exists in the world